roaring/roaring.go

```diff
@@ -98,8 +98,12 @@ type Containers interface {
 
 	Count() uint64
 
-	//Reset will clear the containers collection to allow for recycling during snapshot
+	// Reset will clear the containers collection to allow for recycling during snapshot
```

Comment:
Idiomatically, I think go comments prefer to drop the helper verbs. "Reset clears the ..."

Reply:
sure, I was just fixing the whitespace anyways